src/DropDownMenu/DropDownMenu.js

```diff
@@ -11,30 +11,35 @@ import keycode from 'keycode';
 import Events from '../utils/events';
 import IconButton from '../IconButton';
 import propTypes from '../utils/propTypes';
+import rtl from '../utils/rtl';
 
 
 function getStyles(props, context) {
   const {disabled} = props;
   const spacing = context.muiTheme.baseTheme.spacing;
   const palette = context.muiTheme.baseTheme.palette;
   const accentColor = context.muiTheme.dropDownMenu.accentColor;
+  const normalizeDirection = rtl(context.muiTheme) || ((style) => {
```

Comment:
I have a workaround for this one and it's not mandate to have this fix for the current branch. Let's close it and let me see how much I can help with the next.